setup.cfg

```diff
@@ -46,7 +46,7 @@ all = django; cookiecutter
 
 [test]
 # py.test options when running `python setup.py test`
-addopts = tests --verbose
```

Comment:
Are you sure about removing `tests` at that point? Don't we need to pass the folder to the pytest-runner?